.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,123 @@
+name: New issue for bug report
+description: Got a glitch, crash, or some other problem? If you're sure your issue is reproducible, feel free to report the issue to Natron's development team.
+title: "(Bug): <title>"
+labels: "type:bug"
+body:
+  # Greeter message
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting a bug! Please first check the boxes below to ensure that your bug report meets our requirements.
+  # Requirements checks
+  - type: checkboxes
+    attributes:
+      options:
+        - label: I'm using the latest version of Natron (not required but recommended)
+        - label: I've restarted Natron and the issue persists
+          required: true
+        - label: I've run Natron via the [command line](https://natron.readthedocs.io/en/rb-2.4/devel/natronexecution.html) and the issue persists
+          required: true
+        - label: I've followed the [contributing guidelines](https://github.com/NatronGitHub/Natron/blob/RB-2.4/CONTRIBUTING.md) to the best of my understanding
+          required: true
+        - label: My issue is not on the [issue tracker](https://github.com/NatronGitHub/Natron/issues?q=is%3Aissue+type%3Abug) and is not a duplicate of a forum thread
+          required: true
+        - label: This bug is reproducible
+          required: true
+        - label: This issue is not a feature request or a pull request
+          required: true
+  # Natron & OS versions
+  - type: input
+    id: natron-version
+    attributes:
+      label: Natron version
+      description: Natron version/commit (they can be retrieved from the about window or with `natron --about`)
+      placeholder: "e.g. Natron 2.4.0"
```

People will understand that this is an example placeholder text, it's a pretty common convention with forms and we don't need to specify that it's an example every time.
```suggestion
      placeholder: "Natron 2.4.0"
```